src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1691,6 +1691,37 @@ public String getVariableNamePattern() {
 					})
 					.serializer(new EnumSerializer<>(Gene.class)));
 		}
+		EnumUtils<EntityRegainHealthEvent.RegainReason> regainReasons = new EnumUtils<>(EntityRegainHealthEvent.RegainReason.class, "heal reasons");
+		Classes.registerClass(new ClassInfo<>(EntityRegainHealthEvent.RegainReason.class, "healreason")
```

Comment:
One last minor thing - you should probably ``import org.bukkit.event.entity.EntityRegainHealthEvent.RegainReason`` directly so you don't need to do ``EntityRegainHealthEvent.RegainReason.x``